steno/src/model/stream/stream.cpp

```diff
@@ -69,8 +70,9 @@ Stream::Stream(std::shared_ptr<Config> config)
         config->dewarpingConfig());
 
     m_mediaThread = std::make_unique<MediaThread>(
-        std::make_unique<OdasAudioSource>(10030, 1000 / fps, 4, audioInputConfig),
-        std::make_unique<RawFileAudioSink>("audio_output.raw"), std::make_unique<OdasPositionSource>(10020),
+        std::make_unique<OdasAudioSource>(10030, 1000 / fps, 5, audioInputConfig),
+        std::make_unique<RawFileAudioSink>(QString(config->appConfig()->value(AppConfig::OUTPUT_FOLDER).toString() + "audio_output.raw").toStdString()),
+        std::make_unique<OdasPositionSource>(10020, 1024),
```

Comment:
It would be nice to have these numbers in variables. It would be easier to understand.